source/guide_laravel.rst

```diff
@@ -0,0 +1,151 @@
+.. highlight:: console
+
+.. author:: Christian Kantelberg <uberlab@mailbox.org>
+
+.. tag:: lang-php
+.. tag:: web
+
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/laravel-logo.png
+      :align: center
+
+##########
+Laravel
+##########
+
+.. tag_list::
+
+Laravel is a web application framework with expressive, elegant syntax. We’ve already laid the foundation — freeing you to create without sweating the small things.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`Domains <web-domains>`
+
+
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+Your web domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Fresh Installation
+======================
+
+``cd`` to your user dir, then get Laravel installer via Composer. Finally create a new project using the installer.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust html]$ composer global require "laravel/installer"
```

Comment:
The prompt for this and the following lines should read `[isabell@stardust isabell]$` since we are in `/var/www/virtual/isabell/`.